config/development.yaml

```diff
@@ -0,0 +1,36 @@
+
+service:
```

Comment:
My comments above apply here, too. To recap, I'd propose something like this: 
```
 service:
     listeners:
         - name: notificationAppA
            consumerConfig:
                 ....
            filter:
                 ....
            action:
                 type: webhook
                 webhookConfig:
                     URL: ...
                     retry: ....   
```

Reply:
Great. I moved them to the three types. I use `deliver` instead of action. Also `type` is a reserved word in golang so I use `method` instead.